config/freeradius3/privacyidea-perl

```diff
@@ -0,0 +1,3 @@
+perl privacyidea-perl {
+    filename = ${modconfdir}/${.:instance}/privacyidea_radius.pm
```

Comment:
We should keep the static path, since this is where we install the pm. (just to avoid flaws in certain non-default setups)

Reply:
fixed in 82d037a